index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+
+    <script src="https://kit.fontawesome.com/ccd237d1ae.js" crossorigin="anonymous"></script>
+
+        <title>Sara Mohamoud</title>
+
+    </head>
+
+    <body>
+        <header>
+
+            <div id="logo">
+
+                <span style=color:#DB939A; class="letter"> S </span>
+                <span style=color:#E6C983; class="letter"> A </span>
+                <span style=color:#6C8647; class="letter"> R </span>
+                <span style=color:#B0988C; class="letter"> A </span>
+
+            </div>
+
+            <nav class="navbar">
+
+                <ul>
+                    <li> <a href=#logo class="nav-link"> Home </a> </li>
+                    <li> <a href=#git-hub class="nav-link"> My Work </a> </li>
```

Comment:
Attributes value should be wrapped in speech marks. Look to resources such as W3Schools if unsure on syntax.
```suggestion
                    <li> <a href="#git-hub" class="nav-link"> My Work </a> </li>
```